### PR TITLE
fix(k8s): do not update autoscaled pool size

### PR DESCRIPTION
**Pull request by Sh4d1**
Signed-off-by: Patrik Cyvoct <pcyvoct@scaleway.com>

### CODE REVIEW

File: scaleway/resource_k8s_pool_beta.go [hunk 1]
```diff
@@ -205,8 +205,10 @@ func resourceScalewayK8SPoolBetaUpdate(d *schema.ResourceData, m interface{}) er
 		updateRequest.MaxSize = scw.Uint32Ptr(uint32(d.Get("max_size").(int)))
 	}
 
-	if d.HasChange("size") {
-		updateRequest.Size = scw.Uint32Ptr(uint32(d.Get("size").(int)))
+	if d.Get("autoscaling").(bool) == false {
```
Comment (QuentinBrosse) [inline scaleway/resource_k8s_pool_beta.go]:
```go
if d.Get("autoscaling").(bool) == false && d.HasChange("size") {
```

Comment (Sh4d1) [reply scaleway/resource_k8s_pool_beta.go]: :sweat_smile:

**Comment by Sh4d1** [comment]
Resolved!

**Comment by Sh4d1** [comment]
@QuentinBrosse tests fixed

**Pull request merged**